Lombiq.Analyzers.PowerShell/Invoke-Analyzer.ps1

```diff
@@ -88,19 +90,29 @@ if ((Get-InstalledModule PSScriptAnalyzer -ErrorAction SilentlyContinue).Version
     }
 }
 
+$analyzerParameters = @{
+    "Settings"              = $SettingsPath.FullName
+    "CustomRulePath"        = Join-Path -Path (Split-Path $MyInvocation.MyCommand.Path -Parent) -ChildPath Rules
+    "RecurseCustomRulePath" = $true
+    "IncludeDefaultRules"   = $true
+    "Fix"                   = $Fix
+}
 $results = Find-Recursively -IncludeFile "*.ps1", "*.psm1", "*.psd1" -ExcludeDirectory node_modules |
-    ? { # Exclude /TestSolutions/Violate-Analyzers.ps1 and /TestSolutions/*/Violate-Analyzers.ps1
+    Where-Object { # Exclude /TestSolutions/Violate-Analyzers.ps1 and /TestSolutions/*/Violate-Analyzers.ps1
         $IncludeTestSolutions -or -not (
-            $_.Name -eq 'Violate-Analyzers.ps1' -and
-            ($_.Directory.Name -eq 'TestSolutions' -or $_.Directory.Parent.Name -eq 'TestSolutions')) } |
-    % { Invoke-ScriptAnalyzer $_.FullName -Settings $SettingsPath.FullName } |
+            $PSItem.Name -eq 'Violate-Analyzers.ps1' -and
+            ($PSItem.Directory.Name -eq 'TestSolutions' -or $PSItem.Directory.Parent.Name -eq 'TestSolutions')) } |
+    ForEach-Object { Invoke-ScriptAnalyzer -Path $PSItem.FullName @analyzerParameters } |
     # Only Warning and above (ignore "Information" type results).
-    ? { $_.Severity -ge [Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.DiagnosticSeverity]::Warning }
+    Where-Object { $PSItem.Severity -ge [Microsoft.Windows.PowerShell.ScriptAnalyzer.Generic.DiagnosticSeverity]::Warning }
 
 foreach ($result in $results)
 {
     $message = $result.RuleName + ": " + $result.Message
     Write-FileError -Path $result.ScriptPath -Line $result.Line -Column $result.Column $message
 }
 
-exit $results.Count
+# Exit code indicates the existence of analyzer violations instead of the number of violations, because exiting with
+# code 5 (when there are 5 violations) changes how MSBuild interprets the results and yields the error MSB3075 instead
+# of MSB3073 for some reason.
+exit $results.Count -eq 0 ? 0 : 1
```

Comment:
This breaks backwards compatibility with Windows Powershell, because that doesn't support the ternary operator.
![image](https://user-images.githubusercontent.com/4007293/203595259-cc790eb7-3cf4-43ba-9eb4-6a9ab1abd23c.png)

The script already returns 0 by default, so you only need an if to exit 1.